python/lsst/ts/externalscripts/maintel/take_twilight_flats_comcam.py

```diff
@@ -246,3 +247,17 @@ async def configure(self, config):
         """Take the sequence of twilight flats twilight flats."""
         self.configure_client()
         await super().configure(config)
+
+        if hasattr(config, "ignore"):
+            # Ignoring only specific MTCS components
+            allowed_ignore = ["mtdome", "mtdometrajectory"]
+            for comp in map(RemoteGroup._remote_name_to_attr_format, config.ignore):
```

Comment:
This is using a "private" method from `RemoteGroup`. Not sure I understand what is going on here. Can you please clarify?